src/Data/TreeDiff/Class.hs

```diff
@@ -109,7 +109,7 @@ import Data.These (These (..))
 -- primitive
 import qualified Data.Primitive as Prim
 
-#if MIN_VERSION_base(4,9,0)
+#if MIN_VERSION_base(4,17,0)
```

Comment:
I'm unsure if I was doing something weird, but this module seems to only exist in base >=4.17, that's why I updated this cpp